internal/cephes/igam.go

```diff
@@ -0,0 +1,378 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+ * Cephes Math Library Release 2.0:  April, 1987
+ * Copyright 1985, 1987 by Stephen L. Moshier
+ * Direct inquiries to 30 Frost Street, Cambridge, MA 02140
+ */
+
+package cephes
+
+import (
+	"math"
+)
+
+/*
+ * Adapted from scipy's cephes igam.c
+ */
+
+/*
```

Comment:
I'm working on cleaning up the ported comments. Was there a consensus on the copyright notice?